File: baseLib/src/androidTest/java/me/ycdev/android/lib/common/internalapi/android/os/UserHandleIATest.java
package me.ycdev.android.lib.common.internalapi.android.os;

import org.junit.Test;
import org.junit.runner.RunWith;

import androidx.test.ext.junit.runners.AndroidJUnit4;

import static org.junit.Assert.assertTrue;

@RunWith(AndroidJUnit4.class)
public class UserHandleIATest {
    @Test
    public void test_myUserId() {
        assertTrue(UserHandleIA.checkReflect_myUserId());
    }

}

File: baseLib/src/androidTest/java/me/ycdev/android/lib/common/internalapi/android/os/UserHandleIATest.kt
package me.ycdev.android.lib.common.internalapi.android.os

import androidx.test.ext.junit.runners.AndroidJUnit4
import org.junit.Assert.assertTrue
import org.junit.Test
import org.junit.runner.RunWith

@RunWith(AndroidJUnit4::class)
class UserHandleIATest {
    @Test
    fun test_myUserId() {
        assertTrue(UserHandleIA.checkReflectMyUserId())
    }
}
